hermes/hermes.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
+
+package main
+
+import (
+	"context"
+	"flag"
+	"sync"
+	"strconv"
+
+	cp "github.com/googleinterns/step224-2020/cloudprober"
+)
+
+// FileOperation is an int used as part of the FileOperation enum within the intent log of Hermes' StateJournal.
+type FileOperation int
+
+// FileOperation enum is used for marking the operation intent within the StateJournal of Hermes.
+// FileOperation has two possible values: CREATE, DELETE.
+const (
+	_                    = iota
```

Comment:
Is this needed? It doesn't appear to be used anywhere. I also think that setting _ as a constant is potentially dangerous since it could prevent error checks such as `if _, ok := SomeFunction() {`

Reply:
When I was reading up on enums, it was implied that this was part of you define them. I've looking into it more and this isn't necessary, and wouldn't be helpful here, so I'm going to remove it.